content/docs/command-reference/import.md

```diff
@@ -38,7 +38,7 @@ working directory with its original file name e.g. `data.txt` (or to a location
 provided with `--out`). An _import `.dvc` file_ is created in the same location
 e.g. `data.txt.dvc` – similar to using `dvc add` after downloading the data.
 
-(ℹ️) DVC won't push data imported from other DVC repos to
+ℹ️ DVC won't push data imported from other DVC repos to
 [remote storage](/doc/command-reference/remote). `dvc pull` will download from
 the original source.
```

Comment:
We can now use admonition blocks (https://github.com/iterative/dvc.org/issues/3382):

```
<admon type="info">

DVC won't push data imported from other DVC repos to
[remote storage](/doc/command-reference/remote). `dvc pull` will download from
the original source.

</admon>
```

Reply:
How do I make the links / code block work? Currently, it looks like this:
<img width="703" alt="image" src="https://user-images.githubusercontent.com/32855442/162177804-6dab42d8-9587-4e86-b970-b186048f7fd7.png">

Reply:
It seems to be that the empty lines must be present:
```
# Doesn't render links
<admon type="info">
DVC won't push data imported from other DVC repos to
[remote storage](/doc/command-reference/remote). `dvc pull` will download from
the original source.
</admon>

# Does render links

<admon type="info">

DVC won't push data imported from other DVC repos to
[remote storage](/doc/command-reference/remote). `dvc pull` will download from
the original source.

</admon>

Reply:
I pushed changes with empty lines and links do render properly

Reply:
Oh.. Thanks @daavoo

Reply:
A question: I don't think that we're using admonition blocks in many parts in the doc. Won't this look inconsistent?

Reply:
The idea is to migrate all docs. 

IMO it's a good strategy to just add an admonition block whenever you happen to stump upon one of the old usages, not sure what the rest of @iterative/docs team think

Reply:
> it's a good strategy to just add an admonition block whenever you happen to stump upon one

Yup that's the idea. Not a huge priority to migrate all at once rn